lib/config.js

```diff
@@ -25,7 +25,13 @@ export function getMergedConfig(dir, home) {
 
 export function getConfig(configType, dir) {
   const configPath = getConfigPath(configType, dir);
-  return readJson(configPath);
+  try {
+    return readJson(configPath);
+  } catch (cause) {
+    const error = new Error('Unable to parse config file ' + configPath);
+    error.cause = cause;
+    throw error;
```

Comment:
```suggestion
    throw new Error('Unable to parse config file ' + configPath, { cause });
```

I think that would give the same result on old Node.js versions, because we don't have the logic to print the `cause` on them?

Reply:
@aduh95